heartbeat/dnsupdate.in

```diff
@@ -14,6 +14,8 @@
 # Parameter defaults
 
 OCF_RESKEY_hostname_default=""
+OCF_RESKEY_cname_default=""
+OCF_RESKEY_type_default=""
```

Comment:
`type` should default to old behaviour and not be required to avoid issues for users upgrading.